contrib/initramfs/hooks/zfsunlock

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+PREREQ="dropbear"
+
+prereqs() {
+    echo "$PREREQ"
+}
+
+case "$1" in
+    prereqs)
+        prereqs
+        exit 0
+    ;;
+esac
+
+. /usr/share/initramfs-tools/hook-functions
+
+if [ -r "${DESTDIR}/sbin/dropbear" ]; then
+	cat >"$DESTDIR"/bin/zfsunlock <<'EOF'
+#!/bin/sh
+while [ ! -e /run/zfs_fs_name ] && [ ! -e /run/zfs_unlock_complete ]; do
+	if [ ! $entry_msg_shown ]; then
+		echo -n "Waiting for zfs init script to complete, press Ctrl-C to exit into shell ..."
+		entry_msg_shown=true
+	fi
+	sleep 1;
+	echo -n "."
```

Comment:
Is the constant "." necessary? It's not a huge problem, but if the system sits for a while, that's going to fill the screen. But this is also what is requiring `echo -n` which is non-POSIX (albeit supported by even things like `dash`).

There are a number of `echo -n` instances that could probably be plain `echo`. I'm not seeing why these various files can't have a newline at the end.

Reply:
removed "."